setup.cfg

```diff
@@ -21,6 +25,7 @@ disable=
   duplicate-code,
   fixme,
   invalid-name,
+  logging-fstring-interpolation,
   logging-format-interpolation,
```

Comment:
Maybe, since you removed `.format()` with this PR, also remove this disable. In the long-term we should move logging to lazy formatting anyways, so the formatting only happens, when the log message is actually printed ;-).

Reply:
I didn't remove all, especially in `github.py` where there are some weird formatting I did not touch them, so this is still needed. I rather not touch those myself.